ldk/javascript/examples/self-test-loop/src/tests/filesystem/index.ts

```diff
@@ -390,3 +391,39 @@ export const testFileStat = (): Promise<boolean> =>
       reject(error);
     }
   });
+
+export const testFileUnzip = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      const dirPath = `${await getTestFolderPath()}`;
+      const zipFilePath = `${dirPath}/test_zip.zip`;
+      const textFilePath = `${dirPath}/test_zip.txt`;
+      const writeMode = 0o755;
+      const writeOperation = WriteOperation.overwrite;
+
+      // Create a zip file
+      await filesystem.writeFile({
```

Comment:
Maybe a naive question but won't the bytes contained in this file have to be arranged appropriately in `.zip` format? Simple putting a file onto the filesystem with a `.zip` extension will suffice?